deepl-connector-demo/src_hd/com/axonivy/connector/deepl/demo/TextTranslate/TextTranslate.xhtml

```diff
@@ -14,7 +14,15 @@
 				<h1>Translate your Text</h1>
 				<div style="display: flex; align-items: flex-end;">
 					<div style="display: flex; flex-direction: column;">
-						<p:outputLabel for="inputText" value="inputText" />
+						<div>
+							<p:selectOneMenu id="language1" value="#{data}"
+								converter="ivy.ListItem" >
+								<f:selectItems value="#{data.languages}" var="language"
+									itemLabel="#{language.name()} [#{language.key()}]" />
+							</p:selectOneMenu>
+							<br/>
+							<p:outputLabel for="inputText" value="Translate from this languages" />
```

Comment:
type: 'this languages' vs 'this language'
... but I need a desk-review for this UI anyway: can't quickly make sense of it :)